migrations/v5.js

```diff
@@ -0,0 +1,127 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getCourse, getComponents, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('adapt-contrib-accordion - v4.0.0 > v5.0.0', async () => {
+  let course, courseAccordionGlobals, accordions;
+  const originalAriaRegion = 'Accordion. Select each button to expand the content.';
+  const newAriaRegion = 'List of expandable sections. Select each button to expand the content.';
```

Comment:
```suggestion
  const newAriaRegion = 'List of expandable sections. Select each button to expand the content.';
  const supportedLayouts = [
    'full-width',
    'half-width',
    'both'
  ];
```
Add variable for support layouts to use later